### ISSUE TITLE
Make Keycloak logginpage A11Y screenreader ready

### ISSUE THREAD

**Issue by db-ot** [opencloud-eu/opencloud-compose#35]
## Userstory

As a user with a disability, I want to be able to use the Keycloak login page without barriers, so that I can sign in independently and confidently using assistive technologies.

## Description

This user story focuses on making the Keycloak login page accessible in compliance with WCAG 2.1 AA and relevant accessibility standards (e.g., BITV, EN 301 549). The goal is to ensure that users with various disabilities — including those using screen readers, keyboard-only navigation, or with visual impairments — can successfully and independently log in.

## Acceptance Criteria

- Website is using lang="en" in header, but content is mixed DE / EN
  - two options: clear translation into default english, or use language-dropdown, translate correctly and show lang tag correct
- Input fields programmatically missing the required attribute or aria-required="true"
  - two options: add required attribute or aria-required="true"
- Input fields visually not identifyable as reqiered
  - add a asterisk * or ( required ) by the label text ( asterisk needs explain-text )
- Low contrast ( 1 : 1.6 ) for focused elements like input field
  - adapt the color / borderstyle like we have in opencloud selfs for focused elements

**Comment by tbsbdr** [comment]
@db-ot could you please add BITV 2.0 and WCAG SC number?
lets discuss what/if it should be upstream contributed.

**Comment by vishal-shetty** [comment]
Has any one these points been fixed in the latest keycloak version please?

**Comment by db-ot** [comment]
@vishal-shetty as far as i can see on demo

1. everything is english and lang="en" in header
4. the contrast for focus is standard blue so now 3.88:1.

best regards

**Comment by db-ot** [comment]
A11Y - https://github.com/opencloud-eu/desktop/issues/821
